onnxruntime/contrib_ops/rocm/bert/skip_layer_norm.cc

```diff
@@ -30,6 +31,8 @@ template <typename T>
 SkipLayerNorm<T>::SkipLayerNorm(const OpKernelInfo& op_kernel_info) : RocmKernel(op_kernel_info) {
   ORT_ENFORCE(op_kernel_info.GetAttr<float>("epsilon", &epsilon_).IsOK());
   ORT_ENFORCE(epsilon_ >= 0);
+  const TransformerOptions* options = TransformerOptions::GetInstance();
+  tuning_ = options->IsTuningEnabled();
```

Comment:
@zhangyaobit I think we shall have some unified tuning options pass from session options. As I chatted with @tianleiwu 
> TransformerOptions is for testing purpose, and not for production.
>
> If you have option needed for production, add a session options (preferred) or a provider option.

Reply:
Yes, we have previously had this discussion: it is never intended for production. Guangyun will add it in session option later.

Reply:
Okay, I can change this part when the session option is down.